app.js

```diff
@@ -10,7 +11,7 @@ import {
   buildAgendaContentFromZitting, ensureVersionedAgendaForZitting
 } from './support/agenda-exporter';
 import { signVersionedBesluitenlijst, publishVersionedBesluitenlijst, ensureVersionedBesluitenLijstForZitting, extractBesluitenLijstContentFromDoc, buildBesluitenLijstForZitting } from './support/besluit-exporter';
```

Comment:
I know, and this answer most of the comments, I didn't want to remove the parts we didn't convert yet, because I'm referring to them when creating a new publication, for example now I'm publishing the notulen and I check what was done before. I plan on making a cleanup of the old code when we have everything converted to the new flow, so with the publication of the notulen

Reply:
Sounds good :+1: